homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -199,7 +199,7 @@ async def async_update_data():
         name=name,
         update_method=async_update_data,
         # Polling interval. Will only be polled if there are subscribers.
-        update_interval=timedelta(seconds=10),
+        update_interval=timedelta(seconds=60),
```

Comment:
This is the update interval for the gateway coordinator.

Reply:
I know. Does it really need need to poll every 10 seconds?

Reply:
Honestly, I'm not sure. In this PR we dealing with miio humidifier so we shoudn't touch gateway code. You can change update interval for humidifiers [here](https://github.com/home-assistant/core/blob/496ba8f77fa85b85ef5475ee408c3019f58aac12/homeassistant/components/xiaomi_miio/__init__.py#L133).

Reply:
You are right, my bad.

Reply:
My intention was to update the polling interval for devices. It will update every 60 is no commands are given. It will also be called when a refresh is demanded. So 60 sec. should be more then enough to update the sensors.